build/configs/qemu/tc_64k/defconfig

```diff
@@ -798,7 +798,7 @@ CONFIG_ENABLE_FREE=y
 # CONFIG_HEAPINFO_USER_GROUP is not set
 # CONFIG_HEAPINFO_USER_GROUP_LIST is not set
 # CONFIG_ENABLE_IRQINFO is not set
-CONFIG_ENABLE_KILL=y
+# CONFIG_ENABLE_KILL is not set
```

Comment:
Why to add this change ? If it is required, add it under a new commit.

Reply:
modify